standards/python.md

```diff
@@ -60,4 +60,4 @@ The difference between pinned versions of dependencies in `requirements.txt`  an
 
 Our applications should use requirements files with pinned versions. For example, [consumerfinance.gov has several requirements files](https://github.com/cfpb/consumerfinance.gov/tree/master/requirements) that pin all relevant versions for deployment, testing, local execution, and other specific needs. Some of these requirements files inherit from each other (e.g., [`base.txt` uses the `-r libraries.txt` directive](https://github.com/cfpb/consumerfinance.gov/blob/master/requirements/base.txt#L3) to include the requirements from that file).
 
-Our libraries should specify dependencies with supported version ranges ([and those ranges should be tested with tox](../tox.ini)) in `setup.py` using [the appropriate setuptools `_requires` keyword](https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-dependencies). For example, our [wagtail-sharing library](https://github.com/cfpb/wagtail-sharing) declares install-time requirements with `install_requires` and testing requirements with `extras_require={'testing': testing_extras}`.
+Our libraries should specify dependencies with supported version ranges. The minimum version in the range and the last available version [should be tested with tox](../tox.ini) and can be major, minor, or match). The maximum version that is expected to break compatibility can be a theoretical future major major. The range should be specified in `setup.py` using [the appropriate setuptools `_requires` keyword](https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-dependencies). For example, our [wagtail-sharing library](https://github.com/cfpb/wagtail-sharing) declares install-time requirements with `install_requires` and testing requirements with `extras_require={'testing': testing_extras}`. 
```

Comment:
What does ["major major"](https://en.wikipedia.org/wiki/Major_Major_Major_Major) mean here?

Also, it looks like the documentation link for setuptools dependencies [has moved](https://setuptools.readthedocs.io/en/latest/userguide/dependency_management.html).

Reply:
> major, minor, or match ...

patch?